source/mainnet/smart-contracts/general/contract-instances.rst

```diff
@@ -152,6 +160,15 @@ reduce cost.
 
 .. _contract-instance-operations:
 
+Limits
+======
+
+When initializing or invoking a smart contract, the following limits are enforced:
+
+- The maximum parameter size is 65535 Bytes.
+- Up to 4 billion log items per execution.
+- The maximum return value size is 4 GigaBytes.
```

Comment:
To be more precise it is u32::MAX bytes. 

I think we should have the limits, but still state that they cannot be reached in practice due to NRG bounds.